ykman-gui/qml/PivSetManagementKeyView.qml

```diff
@@ -205,14 +182,47 @@ ColumnLayout {
                     ToolTip.delay: 1000
                     ToolTip.visible: hovered
                     ToolTip.text: qsTr("Management key must be exactly 48 hexadecimal digits.")
-                    visible: hasNewManagementKeyInput
                 }
                 CustomButton {
                     id: randomManagementKeyBtn
                     text: qsTr("Generate")
                     Layout.alignment: Qt.AlignHCenter | Qt.AlignVCenter
                     onClicked: generateManagementKey()
-                    visible: hasNewManagementKeyInput
+                }
+
+                Label {}
+                CheckBox {
+                    id: storeManagementKeyCheckbox
+                    checked: true
+                    text: qsTr("Store on YubiKey, protected by PIN")
```

Comment:
What about a shorter name "Protect with PIN", and then a tool-tip explaining that the key is stored on the yubikey?
(assuming tool-tips on checkboxes work as expected..)